Node installation

jhonhquintero opened the pull request.

Commit:
readme: Add link to how_to. Add how to file.

Commit:
legacy: Add underscore var.

alboyadjian commented on node_modules_locally.md:
```diff
@@ -0,0 +1,8 @@
+- mkdir ~/.npm-packages`
+4. Add this line to .bashrc:
+- PATH=$PATH:$HOME/.npm-packages/bin # Ensure user-global npm binaries can execute.`
+
```
@eknelson17  Could you please add details of how you got Sublime configured to work with the local copy?

alboyadjian replied:
@mr-sudaca  Please also add instrux for installing the eslint-config-airbnb module from OUR repo vs from official repos.  Explain the benefits of doing this, and these steps (w/ some extra explanation)

```
npm -g install eslint
git clone git@github.com:vhl/javascript-style-guide
cd javascript-style-guide/packages/eslint-config-airbnb
npm -g install
```

eknelson17 replied:
@vhladama I added the process that finally worked.

dreampopclub commented:
This looks good to me. Merge?

eknelson17 committed:
Add instructions for Sublime Text.

Probably more detailed than absolutely necessary, but I'm hoping it'll save others some time Googling and/or asking for help.

Also, cleaned up a little Markdown syntax.

Commit:
Add instrux for VIM.

Commit:
Add  'intrux' for emacs.

Commit:
Fix vim instrux.
